docker-compose-dev.yml

```diff
@@ -0,0 +1,20 @@
+version: "3.8"
```

Comment:
We don't need to have this line. It's deprecated now. Please just start with services.

Reply:
done